xExtension-YouTube/extension.php

```diff
@@ -180,11 +200,14 @@ public function getHtmlPeerTubeContentForLink(FreshRSS_Entry $entry, string $lin
 	public function getHtml(FreshRSS_Entry $entry, string $url): string
 	{
 		$content = '';
+	
+		if ($this->autoSize) {
```

Comment:
You removed the class `youtube-plugin-video`. Can you re-add it, to not break existing customizations?